site/viamguides/src/pomodoro-bot/pomodoro-bot.md

```diff
@@ -0,0 +1,439 @@
+   ![select platform](assets/selectPlatform.png)
+1. Use the `viam-agent` to download and install `viam-server` on your Raspberry Pi. Follow the instructions to run the command provided in the setup instructions from the SSH prompt of your Raspberry Pi.
+   ![installation agent](assets/installAgent.png)
+   The setup page will indicate when the machine is successfully connected.
+
+### Add your Raspberry Pi
+
+1. In [the Viam app](https://app.viam.com/fleet/locations), find the **CONFIGURE** tab. It's time to configure your hardware.
+1. Click the **+** icon in the left-hand menu and select **Component**.
+   ![select component](assets/selectComponent.png)
+1. Select `board`, and find the `pi5` module. This adds the module for working with the Raspberry Pi 5's GPIO pins.
+![select board](assets/selectBoard.png)
+1. Notice adding this module adds the board hardware component called `PI5`. The collapsible card on the right corresponds to the part listed in the left sidebar.
+   ![added board](assets/addedBoard.png)
+1. Click **Save** in the top right to save and apply your configuration changes.
+   > aside negative
+   > If any problems occur, check under the **LOGS** tab to see what might be going wrong.
+
+<!-- ------------------------ -->
+## Integrate Google Calendar
+
+Duration: 5
+
+Now let's set up your Pomodoro Bot to work with Google Calendar, so you can stay on top of your schedule without losing focus. Once integrated, your bot will automatically sync with your calendar to manage events and reminders.
+
+### Set Up a Google Cloud Project
+
+1. Go to the [Google Cloud Console](https://console.cloud.google.com/).
+2. Create a new project specifically for this integration.
+   ![new project](assets/googleCloud.png)
+3. Once the project is ready:
+    - Open the [API Library](https://cloud.google.com/apis/docs/overview).
+    - Search for "Google Calendar API".
+      ![calendar API](assets/calendarAPI.png)
+    - Click "Enable" to activate the API.
+      ![enable API](assets/enableAPI.png)
+   
+
+### Create a Service Account
+
+1. In the Google Cloud Console, navigate to **IAM & Admin** > **Service Accounts**.
+![IAM & Admin](assets/IAM.png)
+
+2. Set up a new service account:
+    * Create a new service account
+      ![Create Service Account](assets/createServiceAccount.png)
+      ![Name Service Account](assets/nameServiceAccount.png)
+    * Assign the role "Editor" or "Owner" to give it the necessary permissions.
+      ![Assign Roles](assets/assignRoles.png)
+    * Create and download the key file in JSON format. This file will be essential later.
+      ![Manage Keys](assets/manageKeys.png)
+      ![Create Key](assets/createKey.png)
+      ![Download Key](assets/downloadKey.png)
+
+    * Keep the file in a secure location.
```

Comment:
Can you please add a step that describes how get the file onto the Raspberry Pi, i.e. using `scp` from your computer to the Pi, and noting the path to be used in the module configuration?